test/controllers/users.js

```diff
@@ -1,9 +1,10 @@
 const chai = require('chai'),
   dictum = require('dictum.js'),
-  server = require('./../app'),
+  server = require('../../app'),
+  userService = require('../../app/services/users'),
   should = chai.should();
 
-describe('/users POST', () => {
+describe('users controller', () => {
```

Comment:
What I meant was to add this line but keep the old one nested inside this describe.